drivers/usb/device/usb_dc_stm32.c

```diff
@@ -132,12 +132,13 @@ static const struct gpio_dt_spec ulpi_reset =
 
 /* We need a minimum size for RX FIFO */
 #define USB_FIFO_RX_MIN 160
```

Comment:
The original PR (#62530) doesn't explain where this number comes from, and doesn't clarify whether it is supposed to be bytes or words. @Desvauxm-st?

Reply:
Judging by #69897, it's probably supposed to be words, updated accordingly.

Reply:
FYI @Desvauxm-st is now enjoying his retirement. @marwaiehm-st will have a look.

Reply:
@arbrauns yes it's supposed to be en words